gui/style.css

```diff
@@ -199,3 +199,30 @@ input{
     bottom: 10px;
     right: 10px;
 }
+#loading-overlay {
```

Comment:
add a new line above this for clean code

Reply:
Done